packages/block-library/src/details/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import metadata from './block.json';
+import save from './save';
+
+const { name, category, attributes } = metadata;
+
+export { metadata, name };
+
+export const settings = {
+	title: __( 'Details' ),
+	category,
+	edit,
+	save,
+	keywords: [ __( 'accordion' ) ],
```

Comment:
Since we now know this shouldn't actually be used for an accordion pattern, perhaps we should remove this keyword?